model/Software/Properties/packageVerificationCodeExcludedFile.md

```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# packageVerificationCodeExcludedFile
+
+## Summary
+
+Files which are excluded from the PackageVerificationCode integrity method.
+
+## Description
+
+Files which are excluded from the PackageVerificationCode integrity method.
```

Comment:
There is need to specify what the string represents: basename, full path, starting from, ... ?

Reply:
@zvr - good idea.
I added some text which should resolve this.